.scripts/travis_run_test.py

```diff
@@ -44,3 +43,7 @@
 with open('full_ctest_output.dat', 'w') as outfile:
     outfile.write(ctestout)
 
```

Comment:
It can be very useful; however, its often exceedingly long. I switch it on when I cant debug something any other way.